modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: CONCEPT
+[id="configuring_persistent_storage_{context}"]
+= Configuring persistent storage
+
+Running cluster monitoring with persistent storage means that your metrics are stored to a persistent volume (PV) and can survive a pod being restarted or recreated. This is ideal if you require your metrics or alerting data to be guarded from data loss. For production environments, it is highly recommended to configure persistent storage. Because of the high IO demands, it is advantageous to use local storage.
+
+[id="persistent-storage-prerequisites"]
+== Persistent storage prerequisites
+
+* Dedicate sufficient local persistent storage to ensure that the disk does not become full. How much storage you need depends on the number of pods.
+
+* Make sure you have a persistent volume (PV) ready to be claimed by the persistent volume claim (PVC), one PV for each replica. Because Prometheus has two replicas and Alertmanager has three replicas, you need five PVs to support the entire monitoring stack. The PVs should be available from the Local Storage Operator. This does not apply if you enable dynamically provisioned storage.
```

Comment:
For the last sentence, what do you think of this reprhase:

`The PVs should be available from the Local Storage Operator; they are unavailable if you enable dynamically provisioned storage.` 

Basically, I think it flows better if it's one sentence, however that one sentence is worded.

Reply:
Good point, but this content is moved and not new, and so changing it is outside the scope of this PR. Leaving as-is